crates/hyperswitch_domain_models/src/payments/payment_attempt.rs

```diff
@@ -162,6 +163,111 @@ pub trait PaymentAttemptInterface {
     ) -> error_stack::Result<i64, errors::StorageError>;
 }
 
+#[cfg(feature = "v2")]
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+pub struct PaymentAttempt {
+    pub payment_id: id_type::GlobalPaymentId,
+    pub merchant_id: id_type::MerchantId,
+    pub status: storage_enums::AttemptStatus,
+    pub net_amount: MinorUnit,
```

Comment:
Use `AmountBreakup` in Domain models, while doing remove field in plain structure.